### DISCUSSION TITLE
Help in installing ArgoCD in Kind Cluster

### DISCUSSION THREAD

**Discussion by Sir-gits-a-lot** [argoproj/argo-cd#14072] [Q&A]
I am trying to install ArgoCD inside a kind cluster, after applying all the yaml files, the challenge I am facing is accessing the UI server. 
I have followed all the installation guide but even then I am not able to access it from browser of host Machine. 
I would also want to make clear that I am doing all this from inside a vm which I have created using Virtual Box. 

How can I access the argo-server UI if I deploy argo in a kind cluster is my question.

Any help is appreciated.

**Comment by asher-lab** [comment]
You can create a loadbalancer or export it as a nodeport service. I am using Kind cluster and it works on my end.  Adding in the image. Click below
![Node Port exposed](https://github.com/asher-lab/RepositoryServer/blob/main/NodePort.png)

**Reply by Sir-gits-a-lot** [reply]
Thanks it worked by port forwarding.
